### DISCUSSION TITLE
action restore group doesnt quite work.

### DISCUSSION THREAD

**Discussion by SajidAhmedHEI** [GAM-team/got-your-back#444] [Q&A]
Hi there, 

Wondering if anyone has sucessfully restored to a group so far? I keep getting errors google.auth.exceptions.RefreshError: ('unauthorized_client: Client is unauthorized to retrieve access tokens using this method, or client not authorized for any of the scopes requested.', {'error': 'unauthorized_client', 'error_description': 'Client is unauthorized to retrieve access tokens using this method, or client not authorized for any of the scopes requested.'})
[27263] Failed to execute script 'gyb' due to unhandled exception!

**Comment by jay0lee** [comment]
It works, yes. Show the command you used.

**Reply by SajidAhmedHEI** [reply]
Hi Jay, 

Thank you for your response, here's what I use. 

./gyb --email groupname@domain.com --action restore-group --local-folder /path to backup --service-account

Restore to another mailbox works.

**Comment by jay0lee** [comment]
Ok, the error is telling you that the service account doesn't have
permission to perform that action.

Run:

gyb --action check-service-account --email ***@***.***

And gyb will walk you through steps to authorize.

Jay

**Reply by SajidAhmedHEI** [reply]
Thanks for your response, followed the steps & authorized. 
Been around 4 hours since the change, but still when I run the command to verify it fails. 

Checking service account DwD for groupname@*****.com...
 Scope: https://mail.google.com/                                     FAIL
 Scope: https://www.googleapis.com/auth/apps.groups.migration        FAIL
 Scope: https://www.googleapis.com/auth/drive.appdata                FAIL
 Scope: https://www.googleapis.com/auth/userinfo.email               FAIL
 
 Passed for an email account.. any time frame for the change to reflect?
